repositories/strategies/multipleChoiceOtherAnswerStrategy.js

```diff
@@ -1,42 +1,69 @@
 const { head, isNil } = require("lodash/fp");
 
-const createOtherAnswer = async (trx, { id }) => {
-  const existingOtherAnswer = await trx("Answers")
-    .where({ parentAnswerId: id })
+const findOtherAnswer = async (trx, parentAnswerId) =>
+  trx("Answers")
+    .where({ parentAnswerId, isDeleted: false })
     .first();
```

Comment:
Can we remove the first? Only ever going to be one option that fulfills both of those conditions?

Reply:
The call to `first` resolves the promise and returns the first result. Without that we'd need to await the result of a select and then get the first element of that array ourselves. I think using first is a cleaner implementation in this case.